### ISSUE TITLE
分类问题不是应该使用loss='binary_crossentropy'吗？为什么你用了loss='mse'？

### ISSUE THREAD

**Issue by foguangcui** [hellojixian/ncf-demo#1]
你的代码是 :model.compile(optimizer='adam', loss='mse', metrics=['mae'])

**Comment by hellojixian** [comment]
这里不是分类问题啊 是一个线性的得分 要获取相似度

**Comment by foguangcui** [comment]
明白了，用户-书的互动数据是一个线性问题，可以理解为一个回归问题模型！
